src/modules/DNS.rb

```diff
@@ -516,6 +516,11 @@ def Import(settings)
     # @return autoinstallation settings
     def Export
       expdns = {}
+
+      # It should be case only for installer (1st stage). When routing
+      # was configured via linuxrc, yast needn't to be aware of it. bnc#956012
+      Read() if !@initialized
+
```

Comment:
BTW! Since `expdns` is not `nil` and the indices are literals, those `Ops.set` below can be autocorrected. I only need to tell Rubocop-YaST to consider `Ops.set`. Yay automation!